site/includes/footer.pug

```diff
@@ -1,8 +1,11 @@
-.spectrum-Site-footerContainer
-  hr.spectrum-Rule.spectrum-Rule--small
+unless process.env.VRT
 
-  div(id="footer_global")
+  .spectrum-Site-footerContainer
 
-  include adobe-globalnav.js
+    hr.spectrum-Rule.spectrum-Rule--small
 
-  script(src="//wwwimages2.adobe.com/etc/beagle/public/globalnav/adobe-globalnav/latest/adobe-globalnav.min.js")
+    div(id="footer_global")
+
+    include adobe-globalnav.js
```

Comment:
might be risky to include that. I believe we don't have fully control over that footer and it might lead to false positives in VRT.

Reply:
Yes, that is why I introduced the VRT mode. When you run spectrum-css dev site like this: `VRT=true gulp dev`. The global footer and side navigation will not be rendered.

Reply:
perfect, that's exactly what I was hoping for.